build.gradle.kts

```diff
@@ -33,8 +33,7 @@ repositories {
     jcenter()
 }
 
-dependencies {
-    detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:${Versions.detekt}")
+dependencies { detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:${Versions.detekt}")
```

Comment:
This format looks really strange 🤔

Reply:
The build complained as well
https://github.com/jensim/mega-manipulator/runs/2482821399?check_suite_focus=true#step:7:1098

Reply:
Pro tip:
There are default Build, Test and PluginRun definitions in the repo that will be recognized by intelliJ if you open the project with IntellJ (Community, Edu, Ulti, all work)
Run the Test-step before you commit ;-)
There's also the `ktlintFormat` gradle task defined, that will automatically format most of the code for you

Reply:
my bad, I'm fixing it, i'll add as pre-commit hook